diff -uNr libzypp-bindings-0.5.9/swig/RepoManager.i libzypp-bindings-0.5.9.new/swig/RepoManager.i
--- libzypp-bindings-0.5.9/swig/RepoManager.i	2011-03-01 20:32:06.000000000 +0800
+++ libzypp-bindings-0.5.9.new/swig/RepoManager.i	2011-10-22 18:40:07.041491229 +0800
@@ -1 +1,13 @@
 %include <zypp/RepoManager.h>
+
+#ifdef SWIGPYTHON
+%extend  zypp::RepoManager{
+    bool loadSolvFile(std::string _solv, std::string _alias)
+    {
+        RepoInfo tmpRepo; 
+        tmpRepo.setAlias(_alias);
+        sat::Pool::instance().addRepoSolv(_solv, tmpRepo);
+        return true;
+    }
+}
+#endif
