--- upstream/swig/CMakeLists.txt	2015-03-21 08:06:55.533211670 +0100
+++ libzypp-bindings/swig/CMakeLists.txt	2015-03-21 09:56:26.310170432 +0100
@@ -11,9 +11,9 @@
 # Allow to build only a subset of language bindings via options
 #
 
-OPTION(BUILD_RUBY_BINDINGS "Build Ruby bindings" ON)
+OPTION(BUILD_RUBY_BINDINGS "Build Ruby bindings" OFF)
 OPTION(BUILD_PYTHON2_BINDINGS "Build Python 2 bindings" ON)
-OPTION(BUILD_PERL5_BINDINGS "Build Perl 5 bindings" ON)
+OPTION(BUILD_PERL5_BINDINGS "Build Perl 5 bindings" OFF)
 
 #
 # Let's see which target languages are available
